src/rviz/viewport_mouse_event.h

```diff
@@ -54,7 +54,7 @@ class ViewportMouseEvent
     : panel(p)
     , viewport(vp)
     , type(e->type())
-#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
```
see original PR

Reply:
e is a QMouseEvent here, not a QWheelEvent. This makes a difference.
I needed to change to 6.0.0 in order to compile with 5.15. See original PR.